test/ngAria/ariaSpec.js

```diff
@@ -957,11 +957,7 @@ describe('$aria', function() {
     they('should not prevent default keyboard action if an interactive $type element' +
       'is nested inside ng-click', nodeBlackList, function(elementType) {
         function createHTML(type) {
-          var html = '<' + type + '>';
-
-          if (type === 'INPUT' || 'TYPE'  === 'A') return html;
-
-          return html + '</' + type + '>';
+          return '<' + type + '></' + type + '>';
```

Comment:
`<input></input>` is not correct, but will work nonetheless :smiley:

Reply:
Yes, I simply removed it because the browsers don't care ;)